src/Commands/ClearPendingTenants.php

```diff
@@ -39,11 +39,11 @@ public function handle()
 
         // Skip the time constraints if the 'all' option is given
         if (! $this->option('all')) {
-            if ($olderThanDays = $this->option('older-days') ?? config('tenancy.pending.older_than_days')) {
+            if ($olderThanDays = $this->option('older-than-days')) {
                 $expirationDate->subDays($olderThanDays);
             }
 
-            if ($olderThanHours = $this->option('older-hours') ?? config('tenancy.pending.older_than_hours')) {
+            if ($olderThanHours = $this->option('older-than-hours')) {
```

Comment:
I think only one or the other should be used, not both.

Reply:
Yeah, makes sense, I'll update it